### PR TITLE
Add jobs to transport released prisoners home

### PR DESCRIPTION
**Pull request by Fzzr**
These are mirrors of the prisoner transport jobs, but pay less.

### CODE REVIEW

**Comment by Fzzr** [comment]
Sure, it's not like the player transported the first prisoners ever. Just means that human governments have outsourced prisoner transport (to and from).

File: data/jobs.txt [hunk 1]
```diff
@@ -2217,6 +2217,61 @@ mission "Prisoners [4]"
...
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+		attributes "frontier" "moon" "military"
+	destination
+		distance 2 10
+		government "Republic" "Free Worlds" "Syndicate" "Neutral"
+	to offer
+		random < 15
+	on complete
+		payment 3000 150
+		dialog "The released prisoners have the pale skin of people who spent years without exposure to sunlight. They gather their meager belongings and discharge paperwork and walk into the spaceport. You collect your payment of <payment>."
```
[the remark this answers is not in the document]
Comment (Fzzr) [reply data/jobs.txt]: Shorter version in e82520c.

**Comment by Amazinite** [comment]
>I find myself thinking that all the jobs should have slightly reduced frequencies now that we have so many of them--and especially as we add more.

I'm pretty sure we've already done that once before. :)

File: data/jobs.txt [hunk 2]
```diff
@@ -2217,6 +2217,61 @@ mission "Prisoners [4]"
 			distance 3 4
 		fleet "pirate raid"
 
+mission "Released Prisoners [0]"
+	name "Released prisoner transport to <planet>"
```
Comment (Fzzr) [inline data/jobs.txt]: @Amazinite what do you think about "Released prisoner" -> "Parolee" for this series of jobs to shorten the names?

File: data/jobs.txt [hunk 3]
```diff
@@ -2808,4 +2863,3 @@ mission "Marauder Hunted"
 		system
 			distance 5 5
 		fleet "Marauder fleet X"
-
```
Comment (Pointedstick) [inline data/jobs.txt]: Looks like your editor deleted the trailing blank line at the end of the file. Can you re-add it?

Comment (Fzzr) [reply data/jobs.txt]: Looks like there were previously two EoF newlines and it trimmed one, yeah.

**Pull request merged**